libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -280,6 +333,21 @@ void AP_DDS_Client::write_nav_sat_fix_topic()
     }
 }
 
+void AP_DDS_Client::write_static_transforms()
+{
+    WITH_SEMAPHORE(csem);
+    if (connected) {
+        ucdrBuffer ub;
+        const uint32_t topic_size = tf2_msgs_msg_TFMessage_size_of_topic(&static_transforms_topic, 0);
+        uxr_prepare_output_stream(&session,reliable_out,topics[2].dw_id,&ub,topic_size);
+        const bool success = tf2_msgs_msg_TFMessage_serialize_topic(&ub, &static_transforms_topic);
+        if (!success) {
+            // TODO sometimes serialization fails on bootup. Determine why.
+            AP_HAL::panic("FATAL: DDS_Client failed to serialize\n");
```

Comment:
if this can happen at runtime then we do not want to panic - it would bring down the vehicle
if you want an assert in SITL we can do that, but we should not kill a real vehicle because of a DDS bug

Reply:
Panic removed in `adad06c5c2fd1ad53618ab3ac7a778a0124d30f2`

It's actually odd why these fail, it's usually only within the first second or so. I commented it out, and can investigate later.